src/view/configuration/configuration.jsx

```diff
@@ -527,8 +558,75 @@ const Configuration = () => {
                             </div>
                           </div> */}
 
-                          <h3>Context</h3>
+                          <h3>Data Collection</h3>
 
+                          <div className="u-gapTop">
+                            <InfoTipLayout tip="Indicates whether data associated with clicks on navigational links, download links, or personalized content should be automatically collected.">
+                              <WrappedField
+                                name={`instances.${index}.clickCollectionEnabled`}
+                                component={Checkbox}
+                                label="Enable click data collection"
+                              />
+                            </InfoTipLayout>
+                          </div>
+                          {values.instances[index].clickCollectionEnabled ? (
+                            <div className="FieldSubset u-gapTop">
+                              <InfoTipLayout tip="Regular expression that qualifies a link URL as a download link.">
+                                <FieldLabel
+                                  labelFor="downloadLinkQualifier"
+                                  label="Download Link Qualifier"
+                                />
+                              </InfoTipLayout>
+                              <div>
+                                <WrappedField
+                                  id="downloadLinkQualifierField"
+                                  name={`instances.${index}.downloadLinkQualifier`}
+                                  component={Textfield}
+                                  componentClassName="u-fieldLong"
+                                />
+                                <Button
+                                  id="downloadLinkQualifierTestButton"
+                                  className="RightButton"
```

Comment:
There are some theory discussions to be had here, but I'd probably recommend using `u-gapLeft` instead of `RightButton` for this case.